File: speech-recognition/ibm-watson-speech-to-text/IbmWatsonSpeechToText.java
// "audio" can be a File or an InputStream object

SpeechToText client = new SpeechToText(username, password);

RecognizeOptions options = new RecognizeOptions.Builder()
    .audio(audio)
    .contentType(HttpMediaType.AUDIO_MP3)
    .build();

SpeechRecognitionResults transcript = client.recognize(options).execute();

File: speech-recognition/ibm-watson-speech-to-text/IbmWatsonSpeechToText.kt
// "audio" can be a File or an InputStream object.

val client = SpeechToText(username, password)

val options = RecognizeOptions.Builder()
    .audio(inputFile)
    .contentType(HttpMediaType.AUDIO_MP3)
    .build()

val transcript = client.recognize(options).execute()
